addons/repair/tests/test_repair.py

```diff
@@ -719,3 +719,7 @@ def test_repair_components_lots_show_in_invoice(self):
         self.assertEqual(len(res), 1, "The invoice should have one line")
         self.assertEqual(res[0]['product_name'], self.product_storable_serial.display_name, "The product name should be the same")
         self.assertEqual(res[0]['lot_name'], quant.lot_id.name, "The lot name should be the same")
+
+    def test_create_repair_order_no_initial_picking_type(self):
+        repair_order = self.env['repair.order'].create({'product_id': self.product_product_5.id})
```

Comment:
I think you can just edit the `create` of the previous test ?
So we avoid a brand new one just for this ?